### PR TITLE
Add '2023 Developer Application' blog post

### PR DESCRIPTION
**Pull request by pronskiy**

### CODE REVIEW

File: source/_posts/2023-09-06-application-form-2023.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+---
+title: 'We Are Looking for Developers to Join the PHP Foundation'
+layout: post
+tags:
+  - team
+author:
+  - name: Roman Pronskiy
+    url: https://twitter.com/pronskiy
+  
+  - name: Nils Alderman
```
Comment (naderman) [inline source/_posts/2023-09-06-application-form-2023.md:11]:
```suggestion
  - name: Nils Aderman
```
:joy:

**Review by naderman — APPROVED** [review]
Looks good apart from my name ;-)

File: source/_posts/2023-09-06-application-form-2023.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+Working for the PHP Foundation offers a lot of freedom, but it also requires discipline. Your work will roughly consist of the following:
+
+* 40% Triaging issues and fixing bugs
+* 30% Working on strategic areas
+* 20% Code reviews
+* 10% Preparing RFCs and developing new language features
+
+As you can see, developing new RFCs is a fairly small part of the job. Most of the work is a typical maintenance of an open source project.
+
+We are a fully distributed team. You can work from any place* and at any time. All communication is asynchronous via GitHub, Slack, and Email. We meet for synchronous calls 1-2 times a month.
```
Comment (beberlei) [inline source/_posts/2023-09-06-application-form-2023.md:88]: there is a star missing here for formatting.

Comment (pronskiy) [reply source/_posts/2023-09-06-application-form-2023.md:88]: This is a footnote remark about sanctioned territories. Should I format it differently?

Comment (beberlei) [reply source/_posts/2023-09-06-application-form-2023.md:88]: aah, i thought it was about making something bold. don't mind me then :D

**Pull request merged**